jetbrains-core/src/software/aws/toolkits/jetbrains/core/executables/ExecutableManager.kt

```diff
@@ -210,7 +244,9 @@ sealed class ExecutableInstance {
         val validationError: String
     ) : ExecutableInstance(), ExecutableWithPath
 
-    class UnresolvedExecutable(val resolutionError: String? = null) : ExecutableInstance()
+    class UnresolvedExecutable(val resolutionError: String? = null) : ExecutableInstance() {
+        override val version: String? = null
```

Comment:
it can't, it's version is null. However, like collapsing `validationError` and `resolutionError` it's helpful because we can do stuff like:
```.getExecutableIfPresent<SamExecutable>().version ?: "UNKNOWN"``` instead of having to determine what type it actually is